The Vela Class Hazardous Extraction Ship, a GOLD aligned mining ship.

Erikafox opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
The Vela-Class Hazardous Extraction Vessel is a mining hull currently in use by the Galactic Optimum Labor Division for orbital mining operations in the frontier, designed to leverage mechs to rapidly mine out rich ore sites, and then return to offload their ore. The Vessel's interior is highly malleable, although is currently configured in a more unofficial set up, with the fore being dedicated to the crew, and the aft including both a small science lab and vault.. This specific example hasn't seen the best of upkeep practices, but is still a massive vessel capable of projecting soft power into any sector it's deployed in. 

Other examples of this hull exchange the hangar pod for something else, such as troop transports, manufacturing plants, gas harvesting machines, or external cargo bays. Once a Vela is out of dock it is unlikely to stay stock for long.

This Vela has a crew compliment of 19 crew, divided thus:
1 Captain
1 Supply Officer
1 GOLD Delegate
2 Flight Controllers, for coordinating operations of the miners.
4 Pilots
1 Mechanic
2 Engineers
2 Minutemen, assigned to guard the vessel
and 5 cadets, officially doing whatever.

Power is supplied by a quad-Pacman assembly, feeding two SMESes. The air harvester on the starboard could allow for on the go refueling, and allows the ship to endure sustained operations or resupply its atmos supply in the event of catastrophic damage. 
A small shield lattice prevents collisions from and asteroids and debris, further servicing the ship's asteroid harvesting nature.
Near the front is an office installed for the GOLD Delegate, with a built in privacy shutter, secretarial desk, and fax machine for contacting home. 
The cargo bay contains the crew's latest supply of food, several pizzas and a few boxes of donk pockets. It may not be up to some people's standards, but the pilots seem to enjoy it. In the event of the crew finding something edible, a kitchen has been erected in the fore hallway. 
Mech 4 is currently under construction by the Mechanic, once it's ready, space will have to be made in one of the bays for it.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

<!-- Tick the box below (put an X instead of a space between the brackets) if you have tested your changes and this is ready for review. Leave unticked if you have yet to test your changes and this is not ready for review. -->
![2022 12 13-19 20 07](https://user-images.githubusercontent.com/94164348/207501239-1505395d-9461-4c4e-b1bc-2ebd072bb4f1.png)
![2022 12 13-19 20 09](https://user-images.githubusercontent.com/94164348/207501248-0762931d-da2e-4d5e-855b-c16b0c6626b3.png)
![dreamseeker_1QTUmRyT4h](https://user-images.githubusercontent.com/94164348/207501269-8a0f5ddb-16c8-4b5f-9921-cf53c11e8d86.png)
![dreamseeker_AAMnigf2tb](https://user-images.githubusercontent.com/94164348/207501272-eb7967d3-86a0-4cd8-bb04-a14bd2ba4970.png)
![dreamseeker_3p5OSOC8Ao](https://user-images.githubusercontent.com/94164348/207501277-68a40641-ea6b-4026-b63f-80d9bebeaad3.png)
![dreamseeker_Vk33GkAywz](https://user-images.githubusercontent.com/94164348/207501279-1693fe6f-92ab-4601-ba78-ba9d3b39cd00.png)
![dreamseeker_3X72hJXKVB](https://user-images.githubusercontent.com/94164348/207501284-98c1b7fe-c5e8-4424-a818-ec384e2dc22d.png)


- [x] Ship lands, ship flies, ship doesn't experience sudden mapper induced decompression, outfits look decent, config is properly set up.
## Why It's Good For The Game
Expanding the minutemen lineup with an something that is not explicitly a warship and or security themed vessel gives more character to the faction. It also adds a physical presence to GOLD, which seems to be kinda important out in the frontier. 
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: A Vela-Class hull has been spotted in the region.
add: Vela config
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Erikafox committed:
adds 3 areas for designating pod hangars on larger ships

Erikafox committed:
Adds the Vela and Vela outfits

Erikafox committed:
mucks around in the code

Erikafox committed:
Update minutemen_vela_hazardous_extraction.json

Erikafox committed:
whoops

Erikafox committed:
buttong

Erikafox committed:
spacepods the musical

Erikafox committed:
god pods suck

Erikafox committed:
mapping

Erikafox committed:
code

Erikafox committed:
work

Erikafox committed:
Merge remote-tracking branch 'upstream/master' into totally-normal-mining-ship-you-can-trust-me-I-work-for-GOLD-there-are-no-weapons-aboard-the-ship-we-are-just-here-to-mine-also-have-you-seen-this-guy

Erikafox committed:
Time to TEST

Erikafox committed:
Update minutemen_vela_hazardous_extraction.dmm

Erikafox committed:
GOLD Delegate, mapping bits

Erikafox committed:
mapper smash linter error

Erikafox committed:
engineering office gets another kirby plant so it's less dead inside

Erikafox committed:
Update minutemen_vela_hazardous_extraction.dmm

Erikafox committed:
Update minutemen_vela_hazardous_extraction.dmm

Erikafox committed:
compile, I demand it

Erikafox committed:
feex

Erikafox committed:
integration test fixes

Erikafox committed:
Update minutemen_vela_hazardous_extraction.dmm

Erikafox committed:
found the problem

ZephyrTFA commented:
Space Pods should not be used in ships due to them being buggy pieces of shit. They are likely going to be removed eventually, unless someone can make them not lag incarnate and also not garbage code

Valorium-RB commented:
As far as GOLD lore and design is concerned, this gets my Seal of Approval as the GOLD Lore Head. Good work.

Erikafox commented:
> Space Pods should not be used in ships due to them being buggy pieces of shit. They are likely going to be removed eventually, unless someone can make them not lag incarnate and also not garbage code

My master plan goes something like
>make cool ships that use pods
>someone with actual coding ability notices that pods are janky
>pods get fixed

I'd try it myself honestly but I'm still futzing around with subtyping and outfit defs. The idea of pods is really cool and it'd be a shame to see them gone because they didn't get used and then got removed for being incredibly bad.

Erikafox committed:
Merge branch 'master' into totally-normal-mining-ship-you-can-trust-me-I-work-for-GOLD-there-are-no-weapons-aboard-the-ship-we-are-just-here-to-mine-also-have-you-seen-this-guy

Erikafox committed:
the unpoddening

Erikafox committed:
Merge remote-tracking branch 'upstream/master' into totally-normal-mining-ship-you-can-trust-me-I-work-for-GOLD-there-are-no-weapons-aboard-the-ship-we-are-just-here-to-mine-also-have-you-seen-this-guy

Erikafox committed:
Update minutemen_vela_hazardous_extraction.dmm

Erikafox committed:
Update minutemen_vela_hazardous_extraction.dmm

Erikafox commented:
Changed the ship to use mechs, screenshots and everything have been updated.

Latentish commented on code/modules/spacepods/construction.dm:
```diff
@@ -1,4 +1,4 @@
-/obj/spacepod/examine(mob/user)
+/obj/spacepod/examine(mob/user) //these output above the spacepod's text block for some reason. Not good.
```
A. Spacepods 😔 
B. Is this with your spacepod specifically or in general. It seems like the latter.

Latentish replied:
If so, I'd recommend removing this as a change and making a bug report and request the "reproduceable" tag.

Erikafox replied:
I must've missed that when I was unpodding the ship. Gonna remove this.

Latentish commented:
![image](https://user-images.githubusercontent.com/55075747/198865449-2e37d2ea-a263-4b66-b401-4abca2077d66.png)
![image](https://user-images.githubusercontent.com/55075747/198865453-64cf3364-a8d8-4f61-a1fb-cc30ac73880e.png)
Area fucky wuckies.

Erikafox committed:
Merge remote-tracking branch 'upstream/master' into totally-normal-mining-ship-you-can-trust-me-I-work-for-GOLD-there-are-no-weapons-aboard-the-ship-we-are-just-here-to-mine-also-have-you-seen-this-guy

Erikafox committed:
more unpodding

Erikafox committed:
I can't believe I missed this last time

Erikafox committed:
Merge remote-tracking branch 'upstream/master' into totally-normal-mining-ship-you-can-trust-me-I-work-for-GOLD-there-are-no-weapons-aboard-the-ship-we-are-just-here-to-mine-also-have-you-seen-this-guy

Erikafox committed:
wooah we're half way there

rework in progress.

Erikafox committed:
woaaah we're still about half way there

Erikafox committed:
emergency I made a fucky commit

Erikafox committed:
Vela 2.0

yeah the pr has been open long enough for me to completely redo the internal layout. Sue me.

Erikafox committed:
almost forgot this

Erikafox committed:
and the art supplies

Erikafox committed:
details and fix

Erikafox committed:
whoops

Erikafox committed:
Update minutemen_vela_hazardous_extraction.dmm

Erikafox committed:
Update minutemen_vela_hazardous_extraction.dmm

Erikafox committed:
two lights

Erikafox committed:
GAHHHH

Erikafox committed:
gah 2: GAHHHHH

Erikafox commented:
I redid the entire interior of the ship because I didn't like the aesthetic of the last version. Screenies and the desc were updated.

Erikafox committed:
Merge branch 'master' into totally-normal-mining-ship-you-can-trust-me-I-work-for-GOLD-there-are-no-weapons-aboard-the-ship-we-are-just-here-to-mine-also-have-you-seen-this-guy

Erikafox commented:
BRANCH NAME CURSED
